examples/smartContracts/index.ts

```diff
@@ -140,7 +140,7 @@ const pollAsyncEvents = async (
     console.log(
       `Deployer Wallet Address: ${
         deployerAccount.address
-      } with balance (candidate, final) = (${deployerAccountBalance?.candidate.rawValue()}, ${deployerAccountBalance?.final.rawValue()})`,
+      } with balance (candidate, final) = (${deployerAccountBalance?.candidate.toString()}, ${deployerAccountBalance?.final.toString()})`,
```

Comment:
maybe it will be nice to use the `toMas()` here ?!

Reply:
sure, done, added